mbs_results/outputs/selective_editing_question_output.py

```diff
@@ -0,0 +1,153 @@
+from importlib import metadata
+
+import pandas as pd
+
+from mbs_results.merge_domain import merge_domain
+from mbs_results.unsorted.selective_editing import (
+    calculate_predicted_value,
+    create_standardising_factor,
+)
+
+
+def create_selective_editing_question_output(
+    df: pd.DataFrame,
+    reference,
+    period,
+    domain,
+    question_no,
+    sic,
+    aux: str,
+    a_weight: str,
+    o_weight: str,
+    g_weight: str,
+    predicted_value: str,
+    imputed_value,
+    adjusted_value: str,
+    output_path: str,
+    period_selected: int,
+):
+    """
+    creates and saves the selective editing question output to the output_path folder
+    saves it as a csv with the same version tag
+
+    Parameters
+    ----------
+    df : pd.DataFrame
+        input data, this uses the output produced by the winsorisation functions
+    output_path : str
+        path to save the output csv. Output file path must be the sharepoint as this
+        path is also used to load needed mapping files (could change to individual arg?)
```

Comment:
path is dependent to the current folder structure in sharepoint ? What is going to happen if it changes ?

Reply:
Realised that I refactored code and it no longer saves the output, instead it returns it. Renaming this variable to be mapping file path instead to avoid this issue